packages/core-mobile/app/screens/bridge/Bridge.tsx

```diff
@@ -655,29 +651,39 @@ const Bridge: FC = () => {
           variant="caption"
           sx={{
             color: '$dangerDark'
-          }}>{`Insufficient balance to cover gas costs.\nPlease add ${
-          currentBlockchain === Blockchain.AVALANCHE
-            ? TokenSymbol.AVAX
-            : currentBlockchain === Blockchain.ETHEREUM
-            ? TokenSymbol.ETH
-            : TokenSymbol.BTC
-        }.`}</Text>
+          }}>{`Insufficient balance to cover gas costs.\nPlease add ${sourceNetwork?.networkToken.symbol}.`}</Text>
       )
 
     return null
   }
 
   const renderSelectSection = (): JSX.Element => {
```

Comment:
not related to this pr but we probably should create a tech debt ticket to simplify this monster of a file

Reply:
yeah.. I also had to do some refactoring for reducing cognitive complexity.

Reply:
I just added more action items to this [ticket](https://ava-labs.atlassian.net/browse/CP-9518) you created.